client/css/style.css

```diff
@@ -2090,7 +2090,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
    For example: user has quit ( ===> L   O   L <=== )  */
 
 #windows .header .topic,
-#chat .message .text,
+#chat .message .text span,
```

Comment:
Actually, it turns out that I can't extract it to its own PR, because the fix relies on the fact that adding the `preview` block is taken out of the `parse` method. So these are inter-dependent, but at least I'll keep the commits separate.